src/sentry/services/hybrid_cloud/organization/__init__.py

```diff
@@ -128,6 +128,11 @@ class RpcOrganizationSummary(RpcModel):
     id: int = -1
     name: str = ""
 
+    def __hash__(self) -> int:
```

Comment:
Do we actually intermingle summary objects and normal ones? Seems like we should avoid that

Reply:
This is unfortunately necessary because of the way our serializers work and there isn't any way around that :////

Like honestly there's so much wrong with the way our serializers work, but I dare not go deep into that in THIS PR afterall.